locations/spiders/ecco.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+import scrapy
+
+from locations.hours import DAYS_EN, DAYS_FULL, OpeningHours
+from locations.items import Feature
+
+
+class EccoSpider(scrapy.Spider):
+    name = "ecco"
+    item_attributes = {"brand": "Ecco", "brand_wikidata": "Q1280255"}
+    start_urls = [
+        "https://se.ecco.com/api/store/search?latitudeMin=-90&longitudeMin=-180&latitudeMax=90&longitudeMax=180"
+    ]
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def parse(self, response):
+        stores = json.loads(response.text)
+        for store in stores:
+            if "ecco" in store["n"].lower():
+                yield scrapy.Request(
+                    url="https://se.ecco.com/api/store/finder/" + store["i"], callback=self.parse_store
+                )
+
+    def parse_store(self, response):
+        store = json.loads(response.text)
```

Comment:
Let's do `response.json()` instead of this pattern.

```suggestion
        store = response.json()
```